network/types.go

```diff
@@ -20,6 +20,10 @@ const (
 	Drop
 )
 
+const (
+	DefaultDataNetwork = "default"
```

Comment:
We could either create a new package that holds this name, or keep it consistent across the `sdk-go` module and `sidecar` packages. I've opted for the second.